### DISCUSSION TITLE
What kind of open source contributors is allowed to use copilot for free?

### DISCUSSION THREAD

**Discussion by Tairraos** [community/community#30418] [Copilot Conversations]
What kind of open source contributors is allowed to use copilot for free?
How many stars does it take? Forks? or Released package downloads?

**Comment by benoit-cty** [comment]
Hello, did you find an answer ?

**Comment by benoit-cty** [comment]
Then answer is there : https://github.com/pricing#i-work-on-open-source-projects-can-i-get-access-to-github-copilot-for-free

It's up to them to decide if you deserve it.

See also https://github.com/orgs/community/discussions/19754
